core/server.go

```diff
@@ -747,11 +759,13 @@ func (s *Server) processSignatoryMessage(msg []byte) {
 
```

Comment:
If resp is 
```
{
    Id : 'abcd-1234',
    Result : nil,
    Error : ""
}
```
Then it will return at line `745` without registering bundle errors and closing the `SigningRequest`'s `doneChan`

Reply:
I think this resp is valid though? If `id` is set and either `result` or `error` is set then it will pass validation